tests/aws/services/lambda_/test_lambda.py

```diff
@@ -201,6 +202,63 @@ def test_large_payloads(self, caplog, create_lambda_function, aws_client):
         assert "FunctionError" not in result
         assert payload == json.load(result["Payload"])
 
+    @markers.aws.validated
+    def test_lambda_large_reponse(self, caplog, create_lambda_function, aws_client):
+        # Set the loglevel to INFO for this test to avoid breaking a CI environment (due to excessive log outputs)
+        caplog.set_level(logging.INFO)
+
+        function_name = f"large_response-{short_uid()}"
+        create_lambda_function(
+            handler_file=TEST_LAMBDA_CUSTOM_RESPONSE_SIZE,
+            func_name=function_name,
+            runtime=Runtime.python3_10,
+        )
+        response_size = 6 * 1024 * 1024  # actually + 100 is the upper limit
+        payload = {"bytenum": response_size}  # 6MB response size
+        result = aws_client.lambda_.invoke(
+            FunctionName=function_name, Payload=to_bytes(json.dumps(payload))
+        )
+        assert "FunctionError" not in result
+        assert "a" * response_size == json.loads(to_str(result["Payload"].read()))
```

Comment:
nit: would be nice to align these with "Simplify Lambda payload parsing" https://github.com/localstack/localstack/pull/10210 (just merged today)

Reply:
`json.load(result["Payload"]))`